great_expectations/core/__init__.py

```diff
@@ -21,22 +21,22 @@
 from .urn import ge_urn
 
 __all__ = [
-    ExpectationConfiguration,
-    ExpectationConfigurationSchema,
-    ExpectationSuite,
-    ExpectationSuiteSchema,
-    expectationSuiteSchema,
-    ExpectationSuiteValidationResult,
-    ExpectationSuiteValidationResultSchema,
-    ExpectationValidationResult,
-    ExpectationValidationResultSchema,
-    expectationSuiteValidationResultSchema,
-    expectationValidationResultSchema,
-    get_metric_kwargs_id,
-    IDDict,
-    RunIdentifier,
-    RunIdentifierSchema,
-    ge_urn,
+    "ExpectationConfiguration",
+    "ExpectationConfigurationSchema",
+    "ExpectationSuite",
+    "ExpectationSuiteSchema",
+    "expectationSuiteSchema",
+    "ExpectationSuiteValidationResult",
+    "ExpectationSuiteValidationResultSchema",
+    "ExpectationValidationResult",
+    "ExpectationValidationResultSchema",
+    "expectationSuiteValidationResultSchema",
+    "expectationValidationResultSchema",
+    "get_metric_kwargs_id",
+    "IDDict",
+    "RunIdentifier",
+    "RunIdentifierSchema",
+    "ge_urn",
```

Comment:
Thoughts on doing `obj.__class__.__name__`? What if we rename any of these items?